### ISSUE TITLE
glm compile error on JUWELS Booster - formatting issue in `INSTALL.md`

### ISSUE THREAD

**Issue by Anton-Le** [ComputationalRadiationPhysics/isaac#139]
I observe the following error message when trying to compile PIConGPU with ISAAC on JWB
```
-- Found JPEG: /p/software/juwelsbooster/stages/2020/software/libjpeg-turbo/2.0.5-GCCcore-9.3.0/lib64/libjpeg.so (found version "80")                        
CMake Error at /p/project/pwfaradiation/lib_gpus/ISAAC/DEV/lib/cmake/ISAAC/ISAACConfig.cmake:173 (find_package):                                             
  By not providing "Findglm.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "glm", but
  CMake did not find one.

  Could not find a package configuration file provided by "glm" with any of
  the following names:

    glmConfig.cmake
    glm-config.cmake

  Add the installation prefix of "glm" to CMAKE_PREFIX_PATH or set "glm_DIR"
  to a directory containing one of the above files.  If "glm" provides a
  separate development package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:368 (find_package)
```

Loaded modules:
```
Currently Loaded Modules:
  1) GCCcore/.9.3.0         (H)    14) mpi-settings/CUDA           27) libpciaccess/.0.16     (H)  40) libjpeg-turbo/.2.0.5          (H)
  2) zlib/.1.2.11           (H)    15) ParaStationMPI/5.4.7-1 (g)  28) X11/20200222                41) Python/3.8.5
  3) binutils/.2.34         (H)    16) bzip2/.1.0.8           (H)  29) Tk/.8.6.10             (H)  42) ICU/.67.1                     (H)
  4) GCC/9.3.0                     17) libreadline/.8.0       (H)  30) GMP/6.2.0                   43) Boost/1.74.0
  5) nvidia-driver/.default (H,g)  18) Tcl/8.6.10                  31) libxslt/.1.1.34        (H)  44) Szip/.2.1.1                   (H)
  6) CUDA/11.0              (g)    19) SQLite/.3.32.3         (H)  32) libffi/.3.3            (H)  45) HDF5/1.10.6
  7) ncurses/.6.2           (H)    20) expat/.2.2.9           (H)  33) libyaml/.0.2.5         (H)  46) imkl/.2020.2.254              (H)
  8) CMake/3.18.0                  21) libpng/.1.6.37         (H)  34) Java/1.8                    47) SciPy-Stack/2020-Python-3.8.5
  9) numactl/2.0.13                22) freetype/.2.10.1       (H)  35) PostgreSQL/12.3             48) nettle/.3.6                   (H)
 10) UCX/1.9.0                     23) gperf/.3.1             (H)  36) protobuf/.3.13.0       (H)  49) libdrm/.2.4.99                (H)
 11) pscom/.5.4-default     (H)    24) util-linux/.2.36       (H)  37) gflags/.2.2.2          (H)  50) LLVM/10.0.1
 12) XZ/.5.2.5              (H)    25) fontconfig/.2.13.92    (H)  38) libspatialindex/.1.9.3 (H)  51) libunwind/.1.4.0              (H)
 13) libxml2/.2.9.10        (H)    26) xorg-macros/.1.19.2    (H)  39) NASM/.2.15.03          (H)  52) OpenGL/2020                   (g)
```

After a more focused inspection of `INSTALL.md` it appears that I have run afoul of the formatting, which did not put `glm` as its own point but mixed it with `MPI`

**Comment by Anton-Le** [comment]
#140 would fix this.

**Comment by PrometheusPi** [comment]
Was closed by #140.
